Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp

```diff
@@ -230,7 +230,7 @@ void WebFullScreenManager::enterFullScreenForElement(WebCore::Element* element,
                 return std::nullopt;
 
             auto* image = renderImage->cachedImage()->image();
-            if (!image || !image->isBitmapImage())
+            if (!image || !image->isBitmapImage() || !image->shouldUseQuickLookForFullscreen())
```

Comment:
I think we can remove `!image->isBitmapImage()` now since `shouldUseQuickLookForFullscreen()` will return false for non `BitmapImage` images.